docs/src/main/sphinx/connector/faker.md

```diff
@@ -317,3 +338,41 @@ CREATE TABLE generator.default.customer (
   group_id INTEGER WITH (allowed_values = ARRAY['10', '32', '81'])
 );
 ```
+
+Faker can also automatically set the `default_limit` table property,
+and the `min`, `max`, `null_probability` column properties, based on statistics
+collected by scanning existing data, like in the following example:
```

Comment:
well.. we should at least link to the docs about stats

https://trino.io/docs/current/sql/analyze.html

Also .. let me send a PR to update the related docs for that stuff

Reply:
That page focuses on statistics tracked by Trino, and it's not really relevant in this context, because here we use statistics collected on the fly, when scanning the data. It's the Faker connector that requests particular statistics from the engine, and this doesn't depend on what statistics are provided by other connectors.

Reply:
oh ... so it goes and reads whatever statistics available from the data source directly ..?

Reply:
Somehow we have to detail where the data comes from and if there is anything that can be done if there isnt any

Reply:
No, it doesn't read statistics from the data source. It asks Trino to compute statistics over the data Trino reads from the data source.

The only thing we could be more explicit about here is that Faker discards the data and only uses the statistics. WDYT?

Reply:
Updated that now.